pithos/plugins/_mpris.py

```diff
@@ -158,20 +189,26 @@ def GetAll(self, interface_name):
             }
         elif interface_name == self.MEDIA_PLAYER2_PLAYER_IFACE:
             return {
-                'PlaybackStatus': self._get_playback_status(),
+                'PlaybackStatus': self._get_playback_status,
                 'LoopStatus': "None",
                 'Rate': dbus.Double(1.0),
                 'Shuffle': False,
-                'Metadata': dbus.Dictionary(self._get_metadata(), signature='sv'),
-                'Volume': dbus.Double(self._get_volume()),
-                'Position': dbus.Int64(self._get_position()),
+                'Metadata': dbus.Dictionary(self._get_metadata, signature='sv'),
+                'Volume': dbus.Double(self._get_volume),
+                'Position': dbus.Int64(self._get_position),
                 'MinimumRate': dbus.Double(1.0),
                 'MaximumRate': dbus.Double(1.0),
-                'CanGoNext': self.window.waiting_for_playlist is not True,
+                 #set CanGoNext, CanPlay and CanPause all to True
+                 #to avoid applets ending up in an inconsistent state
+                 #some applets only check this prop once
+                 #if we can play/pause or skip a song will be decided in the methods
+                'CanGoNext': True,
                 'CanGoPrevious': False,
-                'CanPlay': self.window.current_song is not None,
-                'CanPause': self.window.current_song is not None,
-                'CanSeek': False,
+                'CanPlay': True,
+                'CanPause': True,
+                #CanSeek has to be True for some sound applets
+                #to show the song position/duration info
+                'CanSeek': True, 
```

Comment:
Well I am not happy about it.. but I suppose we can. You will need to actually implement `Seek()` and `SetPosition()` though (it would do nothing, but we shouldn't error when it is called...)